bin/strip_logs.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Strip down logs to important information
+ *
+ * Intended to be useful in conjunction with "fetch_papertrail_logs.sh"
+ *
+ * Usage:
+ *  npm run exc ./bin/strip_logs.ts < cat FILENAME
+ *
+ * Accepts no arguments, input taken from stdin.
+ * Use pipe (|) or redirection (<) to provide input.
+ */
+import { evolve, compose, omit, pick } from 'ramda';
+import { Stream } from 'stream';
+
+
+const readStreamToMem = (s: Stream): Promise<string> =>
```

Comment:
Does this use `Stream` because we are piping the data file?

Reply:
Yes, `stdin` is a `ReadStream`